src/createServer.js

```diff
@@ -1,3 +1,86 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => http.createServer((req, res) => {
+  const myURL = new URL(req.url, `http://${req.headers.host}`);
+  const originalText = myURL.pathname.slice(1);
+  const searchParams = Object.fromEntries(myURL.searchParams.entries());
+  const targetCase = searchParams.toCase;
+  const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errorTypes = {
+    textIsMissing: 'text is missing',
+    toCaseIsMissing: 'to case is missing',
+    notSupportedCase: 'not supported case',
+  };
+  const errorsObj = {
+    errors: [],
+  };
+
+  const getError = (errorType) => {
```

Comment:
create new file destructuring with this functions
also, you can create object instead of function:
```
const errorMap = {
  errorType: errorObj
}
```
and get an error from object errorMap.errorType